<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ctkDICOMHostMainWidget</class>
 <widget class="QWidget" name="ctkDICOMHostMainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>887</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DICOM Hosts</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="ctkDICOMAppWidget" name="placeHolderForDICOMAppWidget" native="true"/>
     </item>
     <item>
      <widget class="ctkHostedAppPlaceholderWidget" name="placeHolderForHostedApp">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>300</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QFrame" name="placeHolderForControls">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cbShowControls">
          <property name="text">
           <string>Show extended controls</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>299</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="configureHostedApp">
       <property name="text">
        <string>Configure Hosted App</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sendDataToHostedApp">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Send Data to Hosted App</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="selection">
       <property name="text">
        <string>nothing selected</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkDICOMAppWidget</class>
   <extends>QWidget</extends>
   <header>ctkDICOMAppWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkHostedAppPlaceholderWidget</class>
   <extends>QFrame</extends>
   <header>ctkHostedAppPlaceholderWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
